draft stacks meeting notes (2026-05-18) (#2043)
WebAssembly/meetings: stack/2026/stack-2026-05-18.md

## Participants

Marat Akhin
Zalim Bashorov
Ezra Cooper
Deepti Gandluri
Daniel Hillerström
Sam Lindley
Thomas Lively
Francis McCabe
Andreas Rossberg
Aleksandra Shefer
Veronika Sirotkina
Ben Titzer
Amy Yin

## Agenda items

Compiling Kotlin Coroutines to WebAssembly with Stack-Switching. (Veronika Sirotkina and Alexander Shefer) Slides are available [here](https://github.com/WebAssembly/meetings/blob/main/stack/2026/Presentations/Kotlin%20Coroutines.pdf).

> This meeting was recorded. The recording will be made available in due course.

Reviewer: ```suggestion
This meeting was recorded. The recording is available [here](https://us02web.zoom.us/rec/share/NKZg5dnGjUumps3_A1Zbg2niymNiBqRx5TqjibcAuBz01fqt4mdPMuT71Y9nzmaI.3C5BQ2sOx9U1x4GP).
```
